FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -795,6 +795,203 @@ paths:
                 $ref: '#/components/schemas/Response'
       security:
       - OAuth2: [fraihmwork.component.write, fraihmwork.component.admin]
+  /monitor/v1/component/schedule/unavailability/add:
```

Comment:
Is there a reason we're using a verb in the URL path for this new unavailability API?  In general, we haven't used `add` or `remove` in the paths in other parts of the public API so that makes this one inconsistent.

My understanding from the previous comments is that this might be an artifact of moving from an endpoint with `{uuid}` working against a single component to this collection endpoint, since we now need to provide a body to the delete endpoint to specify what to delete (and the HTTP spec seems to allow a body for a DELETE method but discourages it).

My vote is to stick to:
- POST /monitor/v1/component/schedule/unavailability for creating or modifying (dropping the /add)
- GET /monitor/v1/component/schedule/unavailability stays the same
- DELETE /monitor/v1/component/{componentUUID}/schedule deletes all schedules for the component
- DELETE /monitor/v1/component/{componentUUID}/schedule/{scheduleUUID} deletes a particlar schedule for the component (this would require adding a UUID to the schedule for concrete reference)

Another option would be to overload the POST /monitor/v1/component/schedule/unavailability endpoint to also support updating (so you could effectively drop specific schedules or all schedules by republishing to the same component with a modified schedule)